driver/xrt/include/accl.hpp

```diff
@@ -303,7 +303,7 @@ class ACCL {
    *                       waitfor; nullptr if run_async is false.
    */
   CCLO *copy(BaseBuffer &srcbuf, BaseBuffer &dstbuf, unsigned int count,
```

Comment:
Wouldn't it make sense to also add the overloaded functions for streaming here as well, like we have for send and recv>

Reply:
Yes, implemented it. However, there is also still #99 that also affects the copy. Not sure if we should wait with merging until this is resolved.

Reply:
Yeah we should probably wait on that, we don’t want broken tests in dev